velocity-engine-core/src/main/java/org/apache/velocity/runtime/VelocimacroManager.java

```diff
@@ -515,15 +516,6 @@ public boolean getFromLibrary()
         }
 
         /**
-         * Returns the node tree for this macro.
-         * @return The node tree for this macro.
-         */
-        public SimpleNode getNodeTree()
```

Comment:
Is not used...and any body was calling, just in the velocity-core context... but I ran the all unit test and there aren't error. 

If you know this is an error, please report me and I will fix it